### ISSUE TITLE
What happened to the "This event took place on ..." blurb

### ISSUE THREAD

**Issue by jaygooby** [dannyhope/uxbrighton#197]
I thought there was a frontmatter setting that made it obvious that an event took place.

**Comment by dannyhope** [comment]
Yes, this ought to exist. Not sure why it’s absent. I’m making sure it’s in the design work I’m currently doing (in #199 )
